helm/flink-kubernetes-operator/templates/rbac.yaml

```diff
@@ -61,26 +59,20 @@ rules:
       - update
       - patch
       - delete
+      - deletecollection
   - apiGroups:
-      - extensions
+      - apps
     resources:
-      - deployments
-      - ingresses
```

Comment:
If we wish to drop support for extensions/v1beta1 API version of Ingress, I think we can also remove the code in `IngressUtils.java:266`. This will make the operator unable to create new Ingress resources for Kubernetes 1.19, but that version has been deprecated for a long time.

Reply:
I would appreciate getting rid of the extensions part because this is making some trouble with our deployment on a recent managed Kubernetes cluster (1.29) in combination with Kyverno policy management. Kyverno is preventing the deployment because the extensions API does not seem to exist anymore.

Reply:
@timsn If this is something you specifically don't need you could simply modify your downstream helm chart. But as long as we have users who need this we cannot simply remove it from the base helm template. We would need a discussion on the ML before that

Reply:
@gyfora thanks for your response. That's something I cannot judge because I know not enough about what the extensions are used for and if they are still in use. I just discovered that they lead to problems with our deployment process (Kyverno policy checks).

If this would brake something I would be totally fine to remove the change from this PR and do the changes downstream on my own.